export const professionalExperience = [
  {
    company: "Tempus AI",
    title: "Senior Clinical Scientist",
    location: "Chicago, IL",
    startDate: "Mar 2019",
    endDate: "Present",
    duties: [
      {
        description:
          "Created a tailored bed file to optimize genomic data analysis and facilitate the identification, classification, and clinical reporting of genetic variants on a large gene panel, resulting in improved accuracy and efficiency of downstream analysis.",
      },
      {
        description:
          "Contributed to the development and submission of the xT CDx assay to the FDA by curating and analyzing gene data to rank evidence per gene, ensuring accurate and effective clinical use",
      },
      {
        description:
          "Played a vital role in the development and application of a new hereditary genetic testing panel, Tempus, xGv3 to identify germline variants in genes associated with hereditary cancer syndromes and incidental findings",
      },
      {
        description:
          "Identified gaps and deficiencies in current workflows and executed new workflows and SOPs to harmonize variant classification across multiple institutions",
      },
      {
        description:
          "Spearheaded the development and implementation of a reportable bed file solution for the Tempus xF assay, allowing for targeted reporting of specific genomic regions and improving the overall quality of data analysis.",
      },
      {
        description:
          "Used R programming language to design and implement a regions of interest bed file for reporting insertions and deletions for Tempus xF assay",
      },
      {
        description:
          "Used R programming language to create, update, and modify curated data to justify the inclusion or exclusion of genomic regions for xT-oncology FDA application",
      },
      {
        description:
          "Acted as the scientific lead in the development of the Tempus xF liquid biopsy panel to sequence and report SNVs and indels in clinically relevant regions of 105 genes plus CNVs and DNA rearrangements in a subset of these genes",
      },
      {
        description:
          "Used R programming language to programmatically access data, wrangle data, analyze data and present data to implement changes in workflow or other internal processes",
      },
    ],
  },
  {
    title: "Clinical Scientist",
    company: "Tempus AI",
    location: "Chicago, IL",
    startDate: "Jun 2017",
    endDate: "Mar 2019",

    duties: [
      {
        description:
          "Evaluated data and analyzed variants for a project that lead to the publication of the results in Nature BioTechnology",
      },
      {
        description:
          "Trained over 20 M.A.s, Ph.D.’s and post-baccalaureate in germline and somatic variant classification",
      },
      {
        description:
          "Worked closely with colleagues to build a team of variant scientists from 2 individuals to over 20 highly skilled variant scientists",
      },
      {
        description:
          "ained proficiency in the R programming language and experience using Python, JavaScript, HTML, and CSS. ",
      },
    ],
  },
  {
    title: "Clinical Knowlege Curation Coordinator",

    company: `Harvard Medical School, Partners Healthcare,
    Brigham and Women's Hospital`,
    location: "Cambridge, MA",
    startDate: "Jun 2015",
    endDate: "Jun 2017",
    duties: [
      {
        description:
          "Trained over 25 M.Ds, M.D./Ph.D.’s, rotating fellows, undergraduates, and post-baccalaureate in constitutional variant classification",
      },
      {
        description:
          "Assessed pathogenicity of variants from large gene panels covering cardiomyopathy, hearing loss, pulmonary disease, and Noonan syndrome",
      },
      {
        description:
          "Evaluated data and assessed variants as a critical member of the Geisinger MyCode project including the 56 genes recommended by the ACMG and 20 additional genes from Geisinger",
      },

      {
        description:
          "Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)",
      },
      {
        description: `Extensive experience with population databases (1000 genomes, ESP, ExAC, gnomAD)
Proficient in use of variant databases including- HGMD, ClinVar, LOVD, Deafness Variation database, Cardiodb, MitoMap, Leiden Muscular Dystrophy database, ARUP and COSMIC`,
      },
    ],
  },
  {
    title: "Medical Writer II",
    company: `EBSCO Information Services, DynaMed`,
    location: "Ipswich, MA",
    startDate: "Jun 2014",
    endDate: "Jun 2015",
    duties: [
      {
        description: `Wrote evidence based summaries of clinical trials aimed at serving as a resource for physicians and other health care professionals at point of care.`,
      },
      {
        description: `Wrote evidence based point of care references for a number of clinically important disease topics`,
      },
      {
        description: `Critically assessed published clinical trials for methodological integrity`,
      },
      {
        description: `Critically assessed published clinical trials for inclusion in Dynamed database in areas such as oncology, cardiology, infectious disease, endocrinology and pediatrics`,
      },
      {
        description: `Applied in depth knowledge of statistical methods to critically appraise clinical trials`,
      },
      {
        description: `Extensively worked in an XML environment to deliver point of care clinical information`,
      },
    ],
  },
];

export const skills = [
  {
    skill: "Disease Curation",
  },
  {
    skill: "Variant Analysis",
  },

  {
    skill: "Variant Curation",
  },
  {
    skill: "Variant Classification",
  },
  {
    skill: "FDA Submissions",
  },
  {
    skill: "In Silco Analysis",
  },
  {
    skill: "Family Studies",
  },
  {
    skill: "Population Databases",
  },
  {
    skill: "ClinVar",
  },
  {
    skill: "R Language",
  },
  {
    skill: "JavaScript",
  },
  {
    skill: "TypeScript",
  },
  {
    skill: "Python",
  },
  {
    skill: "HTML",
  },
  {
    skill: "CSS",
  },
  {
    skill: "GitHub",
  },
  {
    skill: "Regex",
  },
  {
    skill: "Data Wrangling",
  },
  {
    skill: "Data Visualization",
  },
  {
    skill: "Data Analysis",
  },
  {
    skill: "Cloud Computing",
  },
];
export const education = [
  {
    institution: "Tufts Graduate School of Biomedical Sciences",
    degree: "Doctor of Philosophy",
    field: "Genetics",
    startDate: "2006",
    endDate: "2014",
    duties: [
      {
        description:
          "Dissertation: Genetic & Genomic Analysis of the mRNA 3's End Processing Subunit Pcf11 Suggests an Expanded Role in Gene Expressio",
      },
      {
        description:
          "Utilized next generation RNA sequencing to generate a large genome wide data set to measure changes in gene expression and potential impacts in cell viability under stress conditions",
      },
      {
        description:
          "Performed bioinformatic analysis of whole genome sequencing data for the purpose of hypothesis testing",
      },
      {
        description:
          "Proficient in the use of programs in the C/C++ environment, PERL script, and use of third party programs for genetic analysis of large data sets",
      },
      {
        description:
          "Coordinated the Genetics program student journal club to provide students exposure to expansive range of genetic research topics",
      },
      {
        description:
          "Supervised junior researchers in laboratory competency and facilitated development of critical-thinking and troubleshooting skills",
      },
    ],
  },
  {
    institution: "University of Massachusetts Boston",
    degree: "Bachelors of Science",
    field: "Biology",
    startDate: "2002",
    endDate: "2006",
    duties: [
      {
        description:
          "Honors Thesis: The function of Rom2 GDP/GTP exchange factor on RHO1 during sporulation in the budding yeast, Saccharomyces Cerevisiae.",
      },
      {
        description:
          "Performed genetic analysis including gene knockout and gene tagging of regulators of cell wall components",
      },
    ],
  },
];

export const publications = [
  {
    title: `Mutations in yeast Pcf11, a conserved protein essential for mRNA 3' end processing and transcription termination, elicit the Environmental Stress Response`,
    year: "2024",
    journal: " Genetics",
    authors:
      "Graber JH, Hoskinson D, Lui H, Micheals KK, Benson PS, Maki NJ, Wilson CL, McGrath C, Puelo F, Pearson E, Kuehner, JN, Moore C",
    edition: `2024 Feb 7;226(2):iyad199. PMID:37967370; PMCID: PMC10847720.`,
    type: "published",
    url: `https://doi.org/10.1093/genetics/iyad199`,
    pdf: "",
  },
  {
    title:
      "Tumor Mutational Burden From Tumor-Only Sequencing Compared With Germline Subtraction From Paired Tumor and Normal Specimens",
    year: "2020",
    journal: "JAMA",
    authors:
      "Parikh K, Huether R, White K, Hoskinson D, Beaubier N, Dong H, Adjei AA, Mansfield AS",
    edition: `2020 Feb 5;3(2):e200202. PMID: 32108894; PMCID: PMC7049088.`,
    type: "published",
    url: `https://doi.org/10.1001/jamanetworkopen.2020.0202`,
    pdf: `https://jamanetwork.com/journals/jamanetworkopen/fullarticle/2762022`,
  },

  {
    title:
      "Integrated genomic profiling expands clinical options for patients with cancer ",
    year: "2020",
    journal: "Nature Biotechnology",
    authors:
      "Beaubier, N., Bontrager, M., Huether, R., Igartua, C., Lau, D., Tell, R., Bobe, A. M., Bush, S., Chang, A. L., Hoskinson, D. C., Khan, A. A., Kudalkar, E., Leibowitz, B. D., Lozachmeur, A., Michuda, J., Parsons, J., Perera, J. F., Salahudeen, A., Shah, K. P., Taxter, T., White, K. P.",
    edition: `37(11), 1351 - 1360`,
    type: "published",
    url: `https://www.nature.com/articles/s41587-019-0259-z`,
    pdf: "",
  },
  {
    title: `Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk`,
    year: "2019",
    journal: "Genetics in Medicine",
    authors:
      "Manickam, K, Buchanan, AH, Schwartz, M, Hallquist, M, Williams, JL, Kulchak, A, Rahm, A, Rocha, H, Savatt, J, Evans, A, Butry, L, Lazzeri, A, Lindbuchler, DM, Flansburg, C, Leeming, R, Vogel, V, Lebo, M, Mason-Suares, H, Hoskinson, DC, Abul-Husn, NS, Dewey, F, Willard, H, Mccormick, CZ, Hartzel, N, Kost, KA, Lavage, DR, Sturm, AC, Frisbie, LR, Person, TN, Metpally, RP, Giovanni, MA, Lowry, LE, Leader, JB, Ritchie, MD, Carey, D, Kirchner, HL, Faucett, W, Williams, S,  Ledbetter, D, Murray M.",
    edition: `7;1(5)`,
    type: "published",
    url: `https://doi.org/10.1038/s41436-019-0560-8`,
    pdf: "",
  },
];

export const resume_basics = {
  title: "Derick Hoskinson, PhD",
  phoneNumber: 312 - 871 - 8067,
  email: "derickchoskinson@gmail.com",
  website: "https://derickchoskinson.com",
  location: "Chicago, IL 60640",
  summary:
    "I am a senior clinical scientist keen on using my expertise in variant curation and analysis to help patients and families. I am also a software developer focused on learning fundamental web technologies such as JavaScript, TypeScript, and React. I am open to new opportunities in the field of variant curation and analysis with a focus on developing web applications to work with genomic data.",
  experience: professionalExperience,
  publications: publications,
  education: education,
  skills: skills,
};
